gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/SubscriptionResource.java

```diff
@@ -57,6 +78,10 @@ public class SubscriptionResource extends AbstractResource {
     @Inject
     private SubscriptionMapper subscriptionMapper;
 
+    @Inject
+    private PlanSearchService planSearchService;
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Can you check if we can use the `GraviteeMapper` instead ?

Reply:
:heavy_check_mark: